src/main/java/nl/novi/techiteasy/dtos/TelevisionInputDto.java

```diff
@@ -0,0 +1,181 @@
+package nl.novi.techiteasy.dtos;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PastOrPresent;
+import jakarta.validation.constraints.PositiveOrZero;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+
+@Component
+public class TelevisionInputDto {
+
+    @NotNull(message = "type may not be empty")
+    private String type;
+    @NotNull(message = "brand may not be empty")
+    private String brand;
+    @NotNull(message = "name may not be empty")
+    private String name;
+    @PositiveOrZero(message = "price has to be zero or a positive value")
+    private double price;
+
+    private double availableSize;
+    private int refreshRate;
+    private String screenType;
+    private String screenQuality;
+
+    @NotNull(message = "Please fill in true or false")
```

Comment:
Deze message geeft niet duidelijk aan welke waarde niet ingevuld is, alleen dat het een boolean is. Je hebt echter 6 booleans in deze dto